### PR TITLE
Labels not displaying in Case Heading

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-2569

### Change description ###
Change `FieldUtils.LABEL_MERGE_FUNCTION` method to make label fields as case values.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by paul-graham — CHANGES_REQUESTED** [review]
Needs an addition to `RELEASE-NOTES.md`.

File: src/shared/services/fields/fields.utils.ts [hunk 1]
```diff
@@ -159,6 +163,10 @@ export class FieldsUtils {
     return fieldValue ? FieldsUtils.currencyPipe.transform(fieldValue / 100, 'GBP', 'symbol') : fieldValue;
   }
 
+  private static getLabel(fieldValue: CaseField) {
```
Comment (phillip-whitaker) [inline src/shared/services/fields/fields.utils.ts:166]: can we have the return type, i.e string eclared.

**Review by phillip-whitaker — CHANGES_REQUESTED** [review]
One minor change requested,.

File: src/shared/services/fields/fields.utils.ts (hunk 1 above)
Comment (kuda1992) [reply src/shared/services/fields/fields.utils.ts:166]: good spot

**Review by phillip-whitaker — DISMISSED** [review]
changes made as requested.

**Review by paul-graham — APPROVED** [review]
Thanks for making the changes, @kuda1992

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Pull request merged**